documentation/book/con-cluster-operator-rbac.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// assembly-operators-cluster-operator.adoc
+
+[id='con-cluster-operator-rbac-{context}']
+= Role-Based Access Control (RBAC)
+
+== Provisioning Role-Based Access Control (RBAC) for the Cluster Operator
```

Comment:
@dmuntima Why do you think this needs to be split? I understand that the module is quite big. But On its own these chapters have no value. There is no procedure, thus is all just explanation of what it does because users are asking for it.

So I can quite easily make it one assembly and 5 concepts if you really want. But I'm missing the added value.

Reply:
@scholzj, to be theoretically correct, we can split this into 5 concepts and an assembly but does not value. Let's leave this as is for now.

Reply:
Ok, good. All other comments should be fixed.